schemas/participant.ts

```diff
@@ -3,18 +3,49 @@ import { z } from 'zod';
 export const participantIdentifierSchema = z
   .string()
   .min(1, { message: 'Identifier cannot be empty' })
-  .max(255, { message: 'Identifier too long. Maxiumum of 255 characters.' });
+  .max(255, { message: 'Identifier too long. Maximum of 255 characters.' })
+  // Ensure that the identifier is not just whitespace or have leading/trailing spaces
+  .refine((identifier) => identifier.trim() === identifier, {
+    message: 'Identifier cannot have leading or trailing spaces',
```

Comment:
This `refine()` checks that the identifier doesn't only consist of spaces, but the message implies that it also detects leading and trailing spaces. The message should say "Identifier requires one or more non-whitespace characters" (or similar).

Its fine to trim leading and trailing whitespace without telling the user. To do this, you need to add `.trim()` to the end of the expression.

Reply:
resolved [7141e0c](https://github.com/complexdatacollective/Fresco/pull/140/commits/7141e0c494fae69109e23f0b2b45482981dbae36)